python/dgl/nn/pytorch/glob.py

```diff
@@ -852,6 +892,8 @@ class PMALayer(nn.Module):
     def __init__(self, k, d_model, num_heads, d_head, d_ff, dropouth=0., dropouta=0.):
         super(PMALayer, self).__init__()
         self.k = k
+        if k == 1:
+            dgl_warning("if k is set to 1, W_q and W_k would not be updated during training.")
```

Comment:
It's not clear what are `W_q` and `W_k` and why they will not be updated in the case of k=1. Perhaps you can include a more detailed note in the doc.

Reply:
DONE